Sources/Danger/GitHubDSL.swift

```diff
@@ -91,17 +92,17 @@ public struct GitHubPR: Decodable {
     /// The users who are assigned to the pull request.
     public let assignees: [GitHubUser]
 
-    /// The ISO6801 date string for when the pull request was created.
-    public let createdAt: String
+    /// The ISO8601 date string for when the pull request was created.
```

Comment:
I got all the comments from `danger-js` so these could use a cleaning up there too 😅 
https://github.com/danger/danger-js/blob/9a501500ee81e593e53062dab1d723908454ac81/source/dsl/Commit.ts#L25